### PR TITLE
 Info about upgrade from Community Version to Enterprise Version

### PR DESCRIPTION
**Pull request by voroyam**
Info about upgrade from Community Version to Enterprise Version

### CODE REVIEW

File: modules/ROOT/pages/faq/index.adoc [hunk 1]
```diff
@@ -1,6 +1,14 @@
 FAQ
 ===
 
+[[I want to upgrade from Community Version to Enterprise Version. What are the changes?]]
+== I want to upgrade from Community Version to Enterprise Version. What are the changes?
+
+In ownCloud Enterprise you will get access to new apps and features, mainly targeted towards enterprises.  Apps and features that ensure security for example.
```
Comment (mmattel) [inline modules/ROOT/pages/faq/index.adoc]: Double blank between:
``enterprises.  Apps``

File: modules/ROOT/pages/faq/index.adoc [hunk 2]
```diff
@@ -1,6 +1,14 @@
...
+
+Your existing *files*, *shares* and *users* remain as they are. You can upgrade to the Enterprise version without concerns.
```
Comment (settermjd) [inline modules/ROOT/pages/faq/index.adoc]: I feel that it reads better when the two sentences are swapped and slightly reworked, as follows:

You can upgrade to the Enterprise version without concern, as your existing *files*, *shares* and *users* remain as they are.

File: modules/ROOT/pages/faq/index.adoc [hunk 3]
```diff
@@ -1,6 +1,14 @@
 FAQ
 ===
 
+[[I want to upgrade from Community Version to Enterprise Version. What are the changes?]]
```
Comment (settermjd) [inline modules/ROOT/pages/faq/index.adoc]: The anchor's quite long, but if you happy with it, then to follow the convention, you need to add hyphens between each word, as in the example below.

File: modules/ROOT/pages/faq/index.adoc (hunk 2 above)
Comment (voroyam) [reply modules/ROOT/pages/faq/index.adoc]: Don't you think that the sentence as it is puts the emphasis on the files shares and users, as they are in the first part of the sentence. In you rewrite it reads better, but the files shares and users are in the last part of the sentence, making the user read.

File: modules/ROOT/pages/faq/index.adoc (hunk 1 above)
Comment (voroyam) [reply modules/ROOT/pages/faq/index.adoc]: good eye

File: modules/ROOT/pages/faq/index.adoc (hunk 3 above)
Comment (voroyam) [reply modules/ROOT/pages/faq/index.adoc]: Do I need this anchor? I just copy pasted from the things that were already there

**Pull request merged**